lib/src/kuzzle.dart

```diff
@@ -31,11 +32,13 @@ class _KuzzleQueuedRequest {
 }
 
 class Kuzzle extends KuzzleEventEmitter {
+
   Kuzzle(
     this.protocol, {
     this.autoQueue = false,
     this.autoReplay = false,
     this.autoResubscribe = true,
+    bool showAPIDeprecatedWarnings = true,
```

Comment:
This seems is a little bit to verbose to me :sweat_smile: A warning is something that is showed and a deprecation come from the API in general
Can we have something simpler like `deprecationWarnings` ?

Reply:
Ping @jenow since you were the one suggesting this name

Reply:
That's the name we have for the JS SDK (deprecationWarning , without the `s`...) and Kuzzle Plugins warnings (`deprecationWarnings`)

Reply:
I agree with @Aschen on this one

Reply:
I do agree too since I was the one that implemented this way for the JS SDK :sweat_smile:

Reply:
> Ping @jenow since you were the one suggesting this name

I was not suggesting this name, I gave you an example with a name like this to show you how it works in Dart x) I don't care what name we put so let's take the same as the JS Sdk

Reply:
It's ok @jenow if we use `deprecationWarnings`?